src/coreComponents/unitTests/CMakeLists.txt

```
@@ -10,3 +10,4 @@ add_subdirectory( finiteVolumeTests )
 add_subdirectory( fileIOTests )
 add_subdirectory( fluidFlowTests )
 add_subdirectory( wellsTests )
+add_subdirectory( physicsSolversTests )
```

```suggestion
add_subdirectory( wavePropagationTests )
```
since we already have some unit tests for physics solvers split between multiple folders (`fluidFlowTests`, `wellTests`, etc)

---

Done